invenio_oaiserver/verbs.py

```diff
@@ -44,6 +46,43 @@ def validate_metadata_prefix(value):
                               field_names=['metadataPrefix'])
 
 
+class DateTime(_DateTime):
+    """DateTime with a permissive deserializer."""
+
+    def from_iso_permissive(datestring, use_dateutil=True):
+        """Parse an ISO8601-formatted datetime and return a datetime object.
+
+        Inspired by the marshmallow.utils.from_iso function, but also accepts
+        datestrings that don't contain the time.
+        """
+        dateutil_available = False
+        try:
```

Comment:
Why are you not using directly `datetime`?
If you want to use `dateutil` maybe it's better to move out the importing.
In that way it can be executed only when python is loaded instead of execute all the times. WDYT?